src/main/java/com/atlassian/bitbucket/jenkins/internal/trigger/BitbucketWebhookConsumer.java

```diff
@@ -78,7 +73,7 @@ void process(MirrorSynchronizedWebhookEvent event) {
                 .collect(Collectors.toSet());
     }
 
-    private static Collection<? extends SCM> getScms(ParameterizedJobMixIn.ParameterizedJob job) {
+    private static Collection<? extends SCM> getScms(ParameterizedJobMixIn.ParameterizedJob<?, ?> job) {
```

Comment:
We can be more specific with our types I think. `<? extends AbstractProject,? extends AbstractBuild>` as we require `AbstractBuild` (and project) it seems like we may just as well be specific about it here, and below